radix-engine/src/vm/wasm/wasmi.rs

```diff
@@ -7,15 +7,14 @@ use wasmi::core::{HostError, Trap};
 use wasmi::errors::InstantiationError;
 use wasmi::*;
 
-use super::InstrumentedCode;
-#[cfg(not(feature = "radix_engine_fuzzing"))]
-use super::MeteredCodeKey;
+pub const DEFAULT_CACHE_SIZE: usize = 1000;
```

Comment:
Previously, we use the same variable for both instrumenter and engine. Had to add it here because instrumenter has been removed.

Also, logically, they represents different concepts.

Reply:
Nice catch! It's a bit messy, in that `wasmer` is importing from `wasmi`. I will move it to "the constants".